### PR TITLE
[joomla] Docker images updates - 2021-08-19

### PR DESCRIPTION
**Pull request by EugeneKorolevich**

### CODE REVIEW

File: versions.yaml [hunk 1]
```diff
@@ -18,22 +18,22 @@
 cloudbuild:
   enable_parallel: false
 versions:
-- dir: 4/php7/debian9/apache
+- dir: 4/php7/debian9/apache/4/
```
Comment (ovk6) [inline versions.yaml:21]: `4/php7/debian9/apache/4.0`

File: versions.yaml [hunk 2]
```diff
@@ -18,22 +18,22 @@
...
+  - 4.0.2-debian9
   - 4.0-debian9
   - 4-debian9
-  - 4.0.0
+  - 4.0.2
   - '4.0'
   - '4'
   - latest
-- dir: 3/php7/debian9/apache
+- dir: 3/php7/debian9/apache/3.10/
```
Comment (ovk6) [inline versions.yaml:36]: remove last /

File: versions.yaml [hunk 3]
```diff
@@ -48,7 +48,7 @@ versions:
   - '3.10'
   - '3'
   - latest
-- dir: 3/php7/debian9/apache
+- dir: 3/php7/debian9/apache/3.9/
```
Comment (ovk6) [inline versions.yaml:51]: remove last /

File: versions.yaml [hunk 4]
```diff
@@ -60,4 +60,5 @@ versions:
   - 3.9-debian9
   - 3.9.28
   - '3.9'
+
```
Comment (ovk6) [inline versions.yaml:63]: remove newline

**Review by ovk6 — CHANGES_REQUESTED** [review]

File: versions.yaml (hunk 1 above)
Comment (EugeneKorolevich) [reply versions.yaml]: done

File: versions.yaml (hunk 2 above)
Comment (EugeneKorolevich) [reply versions.yaml]: done

File: versions.yaml (hunk 3 above)
Comment (EugeneKorolevich) [reply versions.yaml]: done

File: versions.yaml (hunk 4 above)
Comment (EugeneKorolevich) [reply versions.yaml]: done

**Comment by armandomiani** [comment]
@googlebot I consent.

**Review by ovk6 — APPROVED** [review]

**Pull request merged**